.github/merge_rules.json

```
@@ -87,7 +80,7 @@
    {
       "name": "Mobile",
       "patterns": ["ios/**", "android/**", "test/mobile/**"],
-      "approved_by": ["linbinyu", "kit1980", "IvanKobzarev", "dreiss"],
+      "approved_by": ["linbinyu", "xta0", "IvanKobzarev", "dreiss", "raziel"],
```

Not sure why you want to remove kit1980 nor add xta0 here
```suggestion
      "approved_by": ["linbinyu", "IvanKobzarev", "dreiss", "raziel"],
```